shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml

```diff
@@ -128,6 +128,47 @@ services:
       timeout: 2s
       retries: 30
 
+  shenyu-rocketmq:
+    image: rocketmqinc/rocketmq:4.3.2
+    container_name: shenyu-rocketmq
+    restart: always
+    depends_on:
+      shenyu-integrated-test-http:
+        condition: service_healthy
+    ports:
+      - "9876:9876"
+    environment:
+      JAVA_OPT_EXT: "-server -Xms1g -Xmx1g"
+    volumes:
+      - ./logs:/root/logs
+    command: sh mqnamesrv
+    networks:
+      shenyu:
+        aliases:
+          -  shenyu-rocketmq
+
+  shenyu-rmqbroker:
+    image: rocketmqinc/rocketmq:4.3.2
```

Comment:
Why use `rocketmqinc/rocketmq`, is it different from `apache/rocketmq` ?

Reply:
There is no difference between them, apache/rocketmq is officially released by apache. rocketmqinc/rocketmq is a private release, maybe apache/rocketmq is better here.

Reply:
Yep, let's change it to apache/rocketmq.

Reply:
ok